src/core/pageContent/meta/PageMeta.tsx

```diff
@@ -43,10 +45,16 @@ export function PageMeta({ page, headComponent: Head }: PageMetaProps) {
       {openGraphDescription && (
         <meta property="og:description" content={openGraphDescription} />
       )}
-      {image && <meta property="og:image" content={image} />}
+      <meta name="twitter:card" content="summary_large_image" />
+      {image && (
+        <>
+          <meta property="og:image" content={image} />
+          <meta property="twitter:image" content={image} />
+        </>
+      )}
       {openGraphType && <meta property="og:type" content={openGraphType} />}
-      {twitterTitle && <meta name="twitter:title" content={twitterTitle} />}
-      {twitterDescription && (
+      {xTitle && <meta name="twitter:title" content={twitterTitle} />}
+      {xDescription && (
         <meta name="twitter:description" content={twitterDescription} />
       )}
```

Comment:
Now I cannot follow. The xTitle is the twitterTitle or the title, but if it is defined, the twitterTitle is shown. Same with the description.

Reply:
fixed